test/integration/Datatoken.test.ts

```diff
@@ -214,6 +215,10 @@ describe('Datatoken tests', async () => {
     assert(dt.block >= blockNumber, 'incorrect value for: block')
     assert(dt.block < blockNumber + 50, 'incorrect value for: block')
     assert(dt.lastPriceValue === '0', 'incorrect value for: lastPriceValue')
+    assert(
+      dt.eventIndex !== null && dt.eventIndex > 0,
```

Comment:
Can we check for the exact value rather than just checking that it's above 0?

Reply:
I can put the exact value, the reason that I put like that is when testing locally multiple times, that value differs.